fdbrpc/HTTP.actor.cpp

```diff
@@ -342,11 +356,15 @@ ACTOR Future<Void> readHTTPData(HTTPData<std::string>* r,
 		*pos = 0;
 
 		// Read until there are at least contentLen bytes available at pos
-		wait(read_fixed_into_string(conn, r->contentLen, &r->content, *pos));
+		wait(read_fixed_into_string(conn, r->contentLen - r->content.size(), &r->content, *pos));
 
 		// There shouldn't be any bytes after content.
-		if (r->content.size() != r->contentLen)
+		if (r->content.size() != r->contentLen) {
+			TraceEvent(SevError, "HTTPContentLengthMismatch")
```

Comment:
For those new trace events, can we do suppressFor(5.0) to print each type of trace event for every five seconds. Also, we may want to change the Severity from SevError to SevWarn.